### PR TITLE
Add cleanup decorator

### PR DESCRIPTION
**Pull request by getjump**
Usage is like:
`
@cleanup(60)
def mute(update: Update, context: CallbackContext):
`

### CODE REVIEW

**Comment by egregors** [comment]
rel #76

File: bot/utils.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+def cleanup(seconds: int):
+    """Remove messages emitted by decorated function"""
+    logger.debug(f"Removing message from bot in {seconds}")
+    def cleanup_decorator(func):
+        logger.debug(func)
+        @wraps(func)
+        def cleanup_wrapper(*args, **kwargs):
+            for arg in args:
+                if isinstance(arg, CallbackContext):
+                    _hook_function(arg.bot, '_message', lambda *args, **kwargs: 1, lambda msg, *args, **kwargs: ( _remove_Message_after_timer(msg, arg, seconds) ) )
```
Comment (C-Pro) [inline bot/utils.py]: Possibly It's a shame for me, but I was unable to parse this line during the first pass :)
Could that be written to be more understandable?

File: bot/utils.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+from functools import wraps
+from telegram import Message, Bot
+from telegram.ext import CallbackContext
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def _hook_message(bot: Bot, callback_after=lambda x: x):
```
Comment (C-Pro) [inline bot/utils.py]: Why do you need default value for `callback_after` here?

**Comment by egregors** [comment]
@getjump nice one! thanks for your contribution 🔥

**Pull request merged**